components/LinkContainer/LinkContainer.tsx

```diff
@@ -1,11 +1,17 @@
-import React from "react";
+interface LinkContainerProps {
+  name: string;
+  description: string;
+  url: string;
+}
```

Comment:
could you please add this in separate interface.ts file? (LinkContainer.interface.ts)

Reply:
The props are part of the component and initially will not be used outside the component, why take them out to a file?